docs/tutorials/projects/light-up.md

```diff
@@ -1,211 +1,211 @@
-3. Line 44 of your <file>lightupbot.py</file> is where you specify the paths to these files.
-If you put them in the same directory as <file>lightupbot.py</file>, you don't need to edit this line.
-4. Save the file.
-
-### Run the code
-
-Now you are ready to test your robot!
-
-From a command line on your computer, navigate to the project directory and run the code with this command:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-python3 lightupbot.py
-```
-
-If the camera detects a person, it will print to the terminal “This is a person!” and turn on the smart plug.
-If it does not find a person, it will write “There’s nobody here” and will turn off the plug.
-
-Try moving in and out of your webcam's field of view.
-You will see your light turn on and off as the robot detects you!
-
-Your terminal output should look like this as your project runs:
-
-```sh {id="terminal-output" class="command-line" data-prompt="$" data-output="2-5"}
-python3 lightupbot.py
-This is a person!
-turning on
-There's nobody here
-turning off
-```
-
-{{% alert title="Info" color= "info" %}}
-
-You can actually detect any object that is listed in the <file>labels.txt</file> (such as a dog or a chair) but for this tutorial, we are detecting a person.
-
-To detect something else with the camera, just change the string "person" on line 69 of <file>lightupbot.py</file> to a different item in the <file>label.txt</file> file.
-
-```python
-if d.class_name.lower() == "person":
-    print("This is a person!")
-    found = True
-```
-
-{{% /alert %}}
-
-## Next Steps
-
-In this tutorial, you learned how to build an object detection robot that turns your lights on using Viam.
-You could use this same concept to build a smart fan that only turns on if you are sitting at your desk working, turn on the lights in your bathroom mirror only when you are in front of the sink, or activate a pet feeder every time your cat looks at the camera.
-
-To turn this robot into a security alert system, try the other tutorial in this series: [Build a Person Detection Security Robot That Sends You a Photo of the Person Stealing Your Chocolates](/tutorials/projects/send-security-photo/).
-
-For more robotics projects, check out our [other tutorials](/tutorials/).
-
-{{< snippet "social.md" >}}
+---
+title: "Use Object Detection to Turn Your Lights On"
+linkTitle: "Turn on Lights with Object Detection"
+weight: 49
+type: "docs"
+description: "How to turn a light on when your webcam sees a person."
+webmSrc: "/tutorials/img/light-up/light-up.webm"
+mp4Src: "/tutorials/img/light-up/light-up.mp4"
+images: ["/tutorials/img/light-up/light-up.gif"]
+videoAlt: "A person sitting at a desk with a computer and light bulb set up in front of her. As she leaves the light turns off, and as she enters the frame, the light turns back on."
+tags: ["camera", "vision", "detector", "python"]
+# Author: Hazal Mestci
+---
+
+{{< alert title="Caution" color="caution" >}}
```

Comment:
This alert should be removed now

Reply:
no, this PR does not address those other breaking changes